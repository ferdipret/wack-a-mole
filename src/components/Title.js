import styled from "styled-components"

export default styled.div`
  color: hsl(16, 25%, 38%);
  font-family: 'Pacifico', cursive;
  font-size: 50px;
  padding: 20px;
  text-align: center;
`
