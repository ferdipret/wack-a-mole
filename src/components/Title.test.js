import React from 'react'
import { shallow } from 'enzyme'
import Title from './Title'

describe("Title", () => {
  const wrapper = shallow(<Title />)
  it("renders properly", () => {
    expect(wrapper).toMatchSnapshot()
  })
})
