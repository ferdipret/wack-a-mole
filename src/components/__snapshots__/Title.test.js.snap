// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders properly 1`] = `
<div
  className="sc-bdVaJa dVOHJM"
/>
`;
